OfflineService/OfflineServiceViewController.swift

```diff
@@ -42,9 +39,11 @@ class OfflineServiceViewController: UITableViewController, OfflineServiceDataSou
     }
     
     private func setupOfflineServiceDataSource() {
-        offlineServiceDataSource = OfflineServiceDataSource()
-        offlineServiceDataSource?.delegate = self
-        offlineServiceDataSource?.startObservingAvailableRegions()
+        OfflineService.shared.register(observer: self)
```

Comment:
It should be possible to configure `OfflineService` with values provided by end user. Currently it's possible to configure either `OfflineService` or `OfflineServiceUser` due to protection level errors.

Reply:
added method for configuring the user